package.json

```diff
@@ -84,7 +84,7 @@
     "@types/react": "18.2.56",
     "@typescript-eslint/eslint-plugin": "5.62.0",
     "@typescript-eslint/parser": "5.62.0",
-    "@vitest/coverage-v8": "1.2.1",
+    "@vitest/coverage-c8": "0.29.8",
```

Comment:
This may look like a typo, but it is intentional!

Reply:
the package seems to be deprecated, maybe we need to find another solution 
https://www.npmjs.com/package/@vitest/coverage-c8

Reply:
Yep, Unfortunately, our version of Vitest is super out-of-date. I'd recommend we update when we going to Vitest 1.x.
  
  
  I did test upgrading to Vitest as part of [this PR](https://github.com/justeattakeaway/pie/pull/1215), but there's too many issues with the package currently in our repo, and we need to wait for a bug to be resolved.